tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/AwsSqsCommon.cs

```diff
@@ -13,14 +14,15 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AWS.SQS
     internal static class AwsSqsCommon
     {
         private const string DatadogAwsSqsServiceName = "aws-sqs";
-        private const string SqsOperationName = "sqs.request";
+        private const string SqsRequestOperationName = "sqs.request";
         private const string SqsServiceName = "SQS";
+        private const string SnsOperationName = "aws.sqs";
         private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(AwsSqsCommon));
 
         internal const string IntegrationName = nameof(Configuration.IntegrationId.AwsSqs);
         internal const IntegrationId IntegrationId = Configuration.IntegrationId.AwsSqs;
 
-        public static Scope CreateScope(Tracer tracer, string operation, out AwsSqsTags tags, ISpanContext parentContext = null)
+        public static Scope CreateScope(Tracer tracer, string operation, out AwsSqsTags tags, ISpanContext parentContext = null, string spanKind = SpanKinds.Client)
```

Comment:
nitpicking - we tend not to put default values (even though not true everywhere I admit, like the null parentContext just before :p ). Here I would have left it as mandatory. But again nit picking, so I'm ok either way.

Reply:
Will update in an upcoming PR that also ignores adding `peer.service` to inbound spans.